charts/loki-stack/values.yaml

```diff
@@ -34,19 +34,31 @@ filebeat:
             - logs_path:
                 logs_path: "/var/log/containers/"
       output.logstash:
-        hosts: ["logstash-loki:5044"]
+        hosts: ["loki-logstash-headless:5044"] # {{name_of_release}}-logstash-headless:5044
```

Comment:
Why not 
```
  service:
    ports:
      - name: "5044"
        port: 5044
        targetPort: 5044
```
on logstash side ?

Reply:
I think it is worth to merge the commits. As I post [here](https://github.com/grafana/helm-charts/issues/175), I am not the only one that can't use the chart to use Loki with filebeat without changing the `output.logstash.hosts` from `["logstash-loki:5044"]` to `["logstash-loki-headless:5044"]`. I think it is not cool for the chart if we can't use it directly?

Reply:
> Why not
> 
> ```
>   service:
>     ports:
>       - name: "5044"
>         port: 5044
>         targetPort: 5044
> ```
> 
> on logstash side ?

Hey, man! Although all the pods is running after I changed the `output.logstash.hosts` from `["logstash-loki:5044"]` to `["logstash-loki-headless:5044"]`, I can not get any labels on the Grafana website. After I edited the values.yaml as you post, then everything works! I don't know why...